roles/remove-node/pre-remove/tasks/main.yml

```diff
@@ -3,9 +3,13 @@
   set_fact:
     nodes_to_drain: []
 
+- name: Make sure jq is installed
+  package:
+    name: jq
+    state: present
+
 - name: remove-node | Identify nodes to drain, ignore non-cluster nodes
   shell: |
-    set -o pipefail
```

Comment:
Due to this removal, the ansible-lint job is failed like
```
roles/remove-node/pre-remove/tasks/main.yml:11: [E306] Shells that use pipes should set the pipefail option
```
Do we need to remove this line?

/cc @oomichi

Reply:
As shown in the error, it seems that it's problematic to keep it. I might be missing something, though

Reply:
Just force bash shell instead of sh